test/binary/bad-too-many-locals.txt

```diff
@@ -0,0 +1,35 @@
+;;; TOOL: run-objdump-gen-wasm
+;;; ARGS: -x
+;;; ERROR: 1
+magic
+version
+section(TYPE) { count[1] function params[0] results[0] }
+section(FUNCTION) { count[1] type[0] }
+section(CODE) {
+  count[1]
+  func {
+    locals [
+      decl_count[2]
+      leb_u32(0xffffffff) i32
```

Comment:
this is a sick tool! To create my test case, I had to take out my hex editor!

Reply:
Thanks! It has been pretty useful for generating invalid tests like these.